Xamarin.PropertyEditing/Properties/Resources.resx

```diff
@@ -1,1047 +1,1057 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Did you mean to reformat this file?

Reply:
No :(

Reply:
@decriptor I used the Windows editor, which did that, I'll push a nicer fix :/

Reply:
I suppose though if that's how we want to do it for this repo a separate PR to "fix" the formatting might be good

Reply:
These are the least changes I could get right now. Even copying from master and adding the others on, messes up :/